pyproject.toml

```diff
@@ -31,14 +31,10 @@ dev = [
     "ops==2.19.4",  # for static type checking test charms
     "pyright==1.1.398",
     "pytest==8.3.4",
-    "pytest-cov==6.0.0",
+    "pytest-cov==5.0.0",
     "requests>=2.32.3",
     "ruff==0.9.6",
 ]
-docs = [
```

Comment:
This wasn't actually being used -- `make docs` uses its own venv handling in `docs/Makefile`.